app/helpers/search_helper.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+module SearchHelper
+  # This helper integrates the app/views/search/form.html.erb input fields with QBP client query parameters
+  # Omits the following QBP parameters from query:
+  # - query[:address][:address_type]
+  # - query[:multiple_birth_indicator]
+  # - query[:birth_order]
+  # - query[:client_last_updated_date]
+  # - query[:client_last_updated_facility]
+
+  FHIR_TO_HL7_SEX = { 'male' => 'M', 'female' => 'F', 'other' => 'U', 'unknown' => 'U' }.freeze
+
+  def sanitize_input(action_params)
```

Comment:
I removed `sanitize_inputs` from search helper and do sanitization in the search controller now. I kept build_query in the helper however since I want all the logic for transforming the forms data into the QBP client input in one place. I do also have a `query_params` function in search controller that ties it all together.